lib/IAMClient.js

```diff
@@ -592,6 +592,38 @@ class VaultClient {
         }, data, options.reqUid, 'application/json');
     }
 
+    /**
+     * A getter for account canonical IDs given a list of account IDs
+     * @param{String[]} userIds - list of account IDs
+     * @param {Object} options - additional arguments
+     * @param {string} options.reqUid - the request UID
+     * @param {werelogs.RequestLogger} [options.logger] - Logger instance
+     * @param{Function} callback - the callback handling the array of objects
+     * and the error, if there is one
+     * @returns{undefined}
+     */
+    getUsersById(userIds, options, callback) {
+        assert(Array.isArray(userIds), 'userIds must be an array');
+        userIds.every(item => assert(typeof item === 'string'),
```

Comment:
should you also assume item is not an empty string here?

Reply:
I think letting the backend handle an empty string is fine. It could be considered a subset of an invalid id being passed.